plugins/source/aws/resources/services/docdb/event_subscriptions_fetch.go

```diff
@@ -0,0 +1,26 @@
+package docdb
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/docdb"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func fetchDocdbEventSubscriptions(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+	svc := c.Services().DocDB
+
+	input := &docdb.DescribeEventSubscriptionsInput{}
```

Comment:
This needs to have a filter so that it only finds subscriptions for docdb

Reply:
Like this: https://github.com/cloudquery/cloudquery/blob/c9c1e4cf0d3c5175799a21829afefd712f322aad/plugins/source/aws/resources/services/neptune/event_subscriptions_fetch.go#L17